srcpkgs/gst-omx/template

```diff
@@ -1,28 +1,14 @@
 # Template file for 'gst-omx'
 pkgname=gst-omx
-version=1.16.2
-revision=2
-build_style=gnu-configure
-configure_args="--disable-examples
- $(vopt_if rpi --with-omx-target=rpi --with-omx-target=generic)"
-hostmakedepends="pkg-config python glib-devel"
+version=1.18.3
+revision=1
+build_style=meson
+configure_args="-Dexamples=disabled -Dtarget=generic"
+hostmakedepends="pkg-config python3 glib-devel"
 makedepends="gst-plugins-base1-devel"
 short_desc="GStreamer OpenMAX IL wrapper plugin (1.x)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-only"
 homepage="https://gstreamer.freedesktop.org"
 distfiles="${homepage}/src/${pkgname}/${pkgname}-${version}.tar.xz"
-checksum=11ed411a2eba75610d72331eeb14ff05e2df28f4fd05cb69225a88bec6d27439
-
-build_options="rpi"
-
-case "$XBPS_TARGET_MACHINE" in
-	armv[67]*) build_options_default="rpi" ;;
-esac
-
-if [ "$build_option_rpi" ]; then
-	makedepends+=" rpi-firmware rpi-userland-devel"
-	export PKG_CONFIG_PATH="$XBPS_CROSS_BASE/opt/vc/lib/pkgconfig"
-	CFLAGS="-I${XBPS_CROSS_BASE}/opt/vc/include/IL"
-	LDFLAGS="-Wl,-rpath=/opt/vc/lib"
```

Comment:
Replace these with:
```
configure_args+=" -Dheader_path=${XBPS_CROSS_BASE}/opt/vc/include/IL"
```
and keep `$(vopt_if rpi -Dtarget=rpi -Dtarget=generic)` in `configure_args`. This may fix rpi target.

Reply:
Anything that links against shared libraries in `/opt/vc/lib` has to set rpath or the linker will not be able to satisfy these dependencies at runtime. Earlier versions of `rpi-userland` shipped with configurations that modified the dynamic linker search paths to avoid this problem, but that is generally the wrong thing to do on musl (because it will clobber any custom configuration added by users) so I dropped it in favor of rpath entries in affected binaries. Thus, unless the `gst-omx` build system sets the rpath as necessary on its own, dropping the `$LDFLAGS` assignment here is bound to cause more breakage, not less.

Reply:
Thanks for the detailed explanation. I overlooked that you added LDFLAGS in a recent commit.

Reply:
Note that the current version of the `gst-omx` package seems to define `$(BRCMEGL_LIBS)` and `$(BRCMEGL_CFLAGS)` from `pkg-config`, but the Makefiles it produces don't actually use these variables. This is confirmed by looking at the dependencies of the `gst-omx` package for `armv7l`, which has no dependency on `rpi-userland` because the plugin does not link against any of the libraries provided by `rpi-userland`.

Unless the move to meson happened to fix linking against these libraries, I think dropping the `rpi` build option is the right call.